redisvl/index/index.py

```diff
@@ -652,6 +655,74 @@ def aggregate(self, *args, **kwargs) -> "AggregateResult":
         except Exception as e:
             raise RedisSearchError(f"Error while aggregating: {str(e)}") from e
 
+    def batch_search(
+        self, queries: List[str], batch_size: int = 100, **query_params
+    ) -> List[List[Dict[str, Any]]]:
+        """Perform a search against the index for multiple queries.
+
+        This method takes a list of queries and returns a list of search results.
+        The results are returned in the same order as the queries.
+
+        Args:
+            queries (List[str]): The queries to search for.
+            batch_size (int, optional): The number of queries to search for at a time.
+                Defaults to 100.
```

Comment:
My gut tells me we should lower this to like 10. I am thinking about RAM available on shards and the payload size that will come back from searches. Don't have any hard data on this but for queries I think we might want to go a bit smaller with the ability for users to try larger at their own risk

Reply:
Totally agree 👍